manifests/dev/offline-vm.yaml.in

```diff
@@ -0,0 +1,15 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: offlinevirtualmachines.kubevirt.io
```

Comment:
In order that our cluster clean-up can detect that this manifest is from us, you will have to add a 

```
label:
  kubevirt.io: ""
```

label.

Also you will need to expliclity delete this manifest in `cluster/clean.sh` to fix the functional tests on CI (it already contains the manifest without tha tag, so it won't be deleted).

Reply:
Fixed